src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -152,11 +152,15 @@ export const NavSearchBar: React.FC<NavSearchBarProps> = ({ venueId }) => {
 
     return relatedEvents
       .filter((event) => {
-        const isRoomEnabled = isDefined(event.room)
-          ? relatedRoomsByTitle[event.room]?.isEnabled ?? true
-          : true;
+        const isEventRoomEnabledAndExists =
```

Comment:
not sure if your chosen name is 100% valid.
confused about `...AndExists`: for me sounds wrong here.

I'd better rename:
`relatedRooms` -> `enabledRelatedRooms`
`relatedRoomsByTitle` -> `enabledRelatedRoomsByTitle`
`isEventRoomEnabledAndExists` -> `isEventRoomEnabled`

Reply:
done a3708be

Reply:
@alexey-petrov Why did you import `reduce` from `lodash` and replace our existing code there? There should be no need to use a helper from lodash, just use the standard built in `.reduce` functionality the language provides.